tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -347,6 +348,19 @@ def test_dashboard_widget_query_returns_selected_aggregate(self):
         assert response.data["widgets"][0]["queries"][0]["selectedAggregate"] is None
         assert response.data["widgets"][2]["queries"][0]["selectedAggregate"] == 1
 
+    def test_dashboard_details_data_returns_permissions(self):
+        dashboard = Dashboard.objects.create(
+            title="Dashboard With Dataset Source",
+            created_by_id=self.user.id,
+            organization=self.organization,
+        )
+        DashboardPermissions.objects.create(dashboard=dashboard, is_creator_only_editable=False)
+        response = self.do_request("get", self.url(dashboard.id))
+
+        assert response.status_code == 200, response.content
+
+        assert "permissions" in response.data
```

Comment:
We should also assert that the permission is set to `False` here if we're validating that field

Reply:
We could also add a test to cover the case where a dashboard permission isn't created (i.e. retuning `None`) so that behaviour is recorded